Lesson 1/code/exercises/exercise2.html

```diff
@@ -41,23 +41,51 @@
         var arr = [];
         
         function countBoolean() {
-            //!!!!!!!!!
-            return 0;
+            var n = 0;
+            
+            arr.forEach(function(val) {

```

Comment:
Good usage of `array.forEach` function (ES 5.1 /ES 2015)
You could also use `array.filter` combined with `array.length`